dracor.odd

```diff
@@ -486,113 +961,1320 @@
 
-                    <moduleRef key="namesdates"
-                        include="event forename genName listEvent listPerson listRelation nameLink person personGrp persName relation surname"/>
+                    <!-- Elements like add, del, ... and the like are very unlikely to be used, I leave the
+                    elements in but kick the attribute classes out to have less attributes to worry about-->
 
-                    <moduleRef key="corpus" include="particDesc"/>
+                    <!-- delete the atttributes used in manuscript, e.g. @hand -->
+                    <classSpec type="atts" ident="att.written" mode="delete"/>
+                    <classSpec type="atts" ident="att.transcriptional" mode="delete"/>
+                    <classSpec type="atts" ident="att.editLike" mode="delete"/>
 
-                    <moduleRef key="linking" include="ab standOff"/>
+                    <!-- other attribute classes removed -->
+                    <classSpec type="atts" ident="att.declaring" mode="delete"/>
+                    <classSpec type="atts" ident="att.declarable" mode="delete"/>
+                    <classSpec type="atts" ident="att.docStatus" mode="delete"/>
 
-                    <moduleRef key="figures" include="figure"/>
 
-                    <moduleRef key="analysis" include=""/>
+                    <!-- remove attribute @subtype globally -->
+                    <classSpec type="atts" ident="att.typed" mode="change">
+                        <attList>
+                            <attDef ident="subtype" mode="delete"/>
+                        </attList>
+                    </classSpec>
 
 
-                    <!-- Change elements -->
 
-                    <!-- Elements -->
+                    <!-- The following list should include all elements that are added because the 
+                    customization is based on TEI drama. Normally we will leave elements as they are 
+                    but don't display them on the frontend if they are not used in DraCor files. We link to the TEI all documentation instead. 
+                    If an element is used but in a more restricted way than the TEI Drama Customization allows it
+                    we will add a constraint with a schematron rule that displays warnings to alert an ecoder that he or she is using
+                    an element in a way that it might not be processed by the API in the expected way. An encoder may still ignore the warning.-->
 
 
                     <!-- ##### -->
                     <!--   A   -->
                     <!-- ##### -->
 
                     <!-- ab  -->
+                    <!-- Restriction: ab should only include strings; 
+                        maybe no need to restrict that but it was in the legacy-ODD -->
                     <elementSpec ident="ab" module="linking" mode="change">
                         <content>
                             <textNode/>
                         </content>
```

Comment:
If we don't have a good reason to restrict it let's not.

Reply:
I restricted that because <ab> was only used in the licensing pattern and only in the function to provide a label. Agree to remove this restriction

Reply:
OK done